src/3D/Camera.h

```diff
@@ -71,16 +81,20 @@ class Camera
 	void handleMouseInput(const SDL_Event&);
 
 	[[nodiscard]] glm::mat4 getRotationMatrix() const;
-
 protected:
+	bool _focused;
+	glm::vec3 _focusedPosition;
+	float _focusedPositionRadius;
 	glm::vec3 _position;
 	glm::vec3 _rotation;
 	glm::vec3 _dv;
 
 	glm::mat4 _projectionMatrix;
 
 	glm::vec3 _velocity;
-	float _movementSpeed;
+	glm::vec3 _movementSpeed;
+	float _maxMovementSpeed = .0005f;
+	glm::vec3 _acceleration = glm::vec3(0.0f);
```

Comment:
Can you put these definitions in the constructor instead?

Reply:
Done